src/filters/index.js

```diff
@@ -26,6 +28,8 @@ export default {
   exists: existsFilter,
   exist: existsFilter,
   fuzzy: fuzzyFilter,
+  geo_bounding_box: geoBoundingBoxFilter,
+  geo_distance: geoDistanceFilter,
```

Comment:
Same goes for `geo_bounding_box`: `geo-bounding-box` and `geoBoundingBox` should also work.